apps/src/templates/studioHomepages/SectionsTable.jsx

```diff
@@ -89,36 +100,56 @@ const SectionsTable = React.createClass({
         sectionCode: React.PropTypes.string.isRequired
       })
     ),
-    isRtl: React.PropTypes.bool.isRequired
+    isRtl: React.PropTypes.bool.isRequired,
+    isTeacher: PropTypes.bool.isRequired,
+  },
+
+  onLeave() {
+    console.log("How do I remove a student from a section?");
   },
```

Comment:
fwiw, we've always had a rule against shipping with `console.log` calls, particularly back when we supported older browsers that faulted on these statements if the dev console wasn't open.